app/_hub/kong-inc/cors/index.md

```diff
@@ -60,41 +60,48 @@ params:
       required: false
       default:
       value_in_examples: ["http://mockbin.com"]
+
       description: |
         List of allowed domains for the `Access-Control-Allow-Origin` header. If you want to allow all origins, add `*` as a single value to this configuration field. The accepted values can either be flat strings or PCRE regexes. **NOTE**: Prior to Kong 0.10.x, this parameter was `config.origin` (note the change in trailing `s`), and only accepted a single value, or the `*` special value.
     - name: methods
       required: false
       default: "`GET, HEAD, PUT, PATCH, POST, DELETE, OPTIONS, TRACE, CONNECT`"
```

Comment:
I have same question about the `one_of` asked here: https://github.com/Kong/docs.konghq.com/pull/2604

https://github.com/Kong/kong-ee/blob/master/kong/plugins/cors/schema.lua#L54

Reply:
> I have same question about the `one_of` asked here: #2604
> 
> https://github.com/Kong/kong-ee/blob/master/kong/plugins/cors/schema.lua#L54

@murillopaula I'm not sure about this one (no pun intended). The default lists all options, and the example lists 2 options, so what is the intent of `one_of ` for this plugin parameter?

Reply:
The intent of the `one_of` is to specify what are the allowed values for the field. In this case, it's only one of: `GET, HEAD, PUT, PATCH, POST, DELETE, OPTIONS, TRACE, CONNECT`.

My question was how we could explicitly communicate the above information to the user. What do you think about doing the same approach as the response-ratelimiting plugin? See my comment at https://github.com/Kong/docs.konghq.com/pull/2617/#discussion_r580996833

The `config.origin` legacy info I think we should remove as well since we're in 2.3.x already...

Reply:
> The intent of the `one_of` is to specify what are the allowed values for the field. In this case, the allowed values can only be: `GET, HEAD, PUT, PATCH, POST, DELETE, OPTIONS, TRACE or CONNECT`. Anything outside this list if not allowed.

@murillopaula the * char is still allowed when configuring the CORS plugin (I had tested it in Konnect), but it is not reflected in the schema. I think we need to mention it.

> My question was how we could explicitly communicate the above information to the user. What do you think about doing the same approach as the response-ratelimiting plugin? See my comment at #2617

Yes agree.

> The `config.origin` legacy info I think we should remove as well since we're in 2.3.x already...

Will do.